varya/assets/sass/variables-editor.scss

```diff
@@ -25,6 +25,8 @@
 	@include quote-variables();
 	@include separator-variables();
 	@include utilities-variables();
+	/* Components */
+	@include header-variables();
```

Comment:
I'm importing the header variables here to get access to the `--primary-nav` variables in the editor. Might be overkill since it also includes a lot of branding variables?

Reply:
I think it's okay, since we also may want to use some of these variables later on in the Site Title + Site Description blocks, etc.